src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -13,10 +16,30 @@ function HeatmapVis(props: Props): JSX.Element {
   const textureData = useMemo(() => computeTextureData(data), [data]);
 
   return (
-    <Canvas orthographic>
-      <ambientLight />
-      {textureData && <Mesh dims={dims} textureData={textureData} />}
-    </Canvas>
+    <AutoSizer>
+      {({ width, height }) => {
+        return (
+          <div className={styles.chart} style={{ width, height }}>
+            <Canvas className={styles.heatmap} orthographic>
+              <ambientLight />
+              {textureData && <Mesh dims={dims} textureData={textureData} />}
```

Comment:
I don't understand how you can do that: `dims` is the shape of the dataset, no ?

Reply:
ah yes :face_palm:

Reply:
I think you need to add a `div` around `Canvas`, because `Canvas` renders a `div` with 100% width and height, and since it's a direct child of `chart`, it takes the width and height of `chart`, ignoring the size of the axis.

Reply:
Ah-ha ! So that was the problem !